codesystems/CodeSystem-UKCore-FundingCategory.xml

```diff
@@ -31,5 +32,33 @@
   <concept>
     <code value="devolved-nations" />
     <display value="Devolved nations" />
+    <concept>
+      <code value="nhs-scotland" />
+      <display value="NHS Scotland" />
+    </concept>
+    <concept>
+      <code value="nhs-wales" />
+      <display value="NHS Wales" />
+    </concept>
+    <concept>
+      <code value="nhs-northern-ireland" />
+      <display value="NHS Northern Ireland" />
```

Comment:
It is HSC for N. Ireland https://www.northerntrust.hscni.net/about-the-trust/trust-overview-2/health-and-social-care-in-northern-ireland/ 

But I'm wondering what do we get from adding the NHS prefix? Could we just have england, scotland, wales, etc?